src/extension/index.ts

```diff
@@ -67,14 +68,28 @@ export async function activate(context: ExtensionContext) {
         update();
     }
 
+    // For now, we keep file watch functionality limited to the API.
+    // After we're sure it's working well, consider also enabling it for files opened manually.
+    const watcher = watch([], {
+        ignoreInitial: true
```

Comment:
Not that it matters since we start out with an empty array, but `ignoreInitial` ensures that the change event doesn't get called when the watcher does the first pass on all the files to ready itself.